group-generators/generators/rhinofi-power-users/index.ts

```diff
@@ -1,6903 +1,53 @@
-
-import { Tags, ValueType, GroupWithData, AccountSource } from "topics/group";
+import { dataProviders } from "@group-generators/helpers/data-providers";
+import {
+  Tags,
+  ValueType,
+  GroupWithData,
+  AccountSource
+} from "topics/group";
 import {
   GenerationContext,
   GenerationFrequency,
   GroupGenerator,
 } from "topics/group-generator";
 
+type UserEntry = {
+  ethAddress: string;
+  active_weeks: number;
+};
+
+export async function formatUsers(context: GenerationContext, data: Array<UserEntry>): Promise<GroupWithData[]> {
```

Comment:
There is an extra indentation on the for

Reply:
Fixed.